app/src/main/java/com/somestupidappproject/betago/game/Game.java

```diff
@@ -61,7 +61,15 @@ public boolean undoMostRecentMove() {
     }
 
     public boolean playStone(Stone stone) {
-        if (BoardUtils.isValidMove(stone, board, isBlacksMove)) {
+        // get the previous captured stones and previously played stone
+        Stone previousPlayedStone = null;
+        ArrayList<Stone> previousCapturedStones = null;
+        if (previousMoves.size() > 0) {
+            previousCapturedStones = new ArrayList<Stone>(){{ addAll(previousMoves.lastElement().getCapturedStones()); }};
+            previousPlayedStone = previousMoves.lastElement().getStone();
+        }
+
+        if (BoardUtils.isValidMove(stone, board, isBlacksMove, previousCapturedStones, previousPlayedStone)) {
```

Comment:
Seems like it makes more sense to pass in the previous Move instead of splitting up previousCapturedStones and previousPlayedStone